css/properties/white-space.json

```
@@ -96,6 +96,54 @@
             }
           }
         },
+        "nowrap": {
+          "__compat": {
+            "description": "<code>nowrap</code>",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": "18"
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "3.5"
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "8"
+              },
+              "opera": {
+                "version_added": "9.5"
+              },
+              "opera_android": {
+                "version_added": "46"
+              },
+              "safari": {
+                "version_added": "3"
+              },
+              "safari_ios": {
+                "version_added": "3"
+              },
+              "samsunginternet_android": {
+                "version_added": "11.2"
+              },
+              "webview_android": {
+                "version_added": "37"
```

Since Android's original browser was WebKit derived, this was also almost certainly supported from the first release:

```suggestion
                "version_added": "1"
```

---

Ah, I see what you mean here. I think the correct thing to do is change both this version to `"1"` and the parent feature to `"1"` (around line 42)

---

Sure